src/Console/WarmupCommand.php

```diff
@@ -18,15 +18,17 @@
 namespace Colopl\Spanner\Console;
 
 use Colopl\Spanner\Connection as SpannerConnection;
+use Google\Cloud\Core\Exception\ServiceException;
 use Illuminate\Console\Command;
 use Illuminate\Database\DatabaseManager;
 
 class WarmupCommand extends Command
 {
     protected $signature = 'spanner:warmup {connections?* : The database connections to be warmed up}
-               {--refresh : Will clear all existing sessions first.}';
+               {--refresh : Will clear all existing sessions first.}
+               {--skip-on-error : Will skip the connection if error is thrown.}';
 
-    protected $description = 'Warmup Spanner\'s Session Pool.';
+    protected $description = "Warmup Spanner's Session Pool.";
 
     public function handle(DatabaseManager $db): void
```

Comment:
What is checked exception? I thought `@throws` is used to notify what the method is capable of throwing, in which case there are way more to document than just `ServiceException`.

Reply:
If this is documented for clarity, I think there needs to be more detailed description about when and why they are thrown, or else it would be too difficult to guess how or when one should catch such a case.